NetSDR_Streaming.cpp

```diff
@@ -178,47 +179,52 @@ int SoapyNetSDR::readStream(
 	long long &timeNs,
 	const long timeoutUs )
 {
-	// fill in the select structures
+	size_t nn = numElems;
+	float *out = (float *)buffs[0];
+
+	// get data from the buffer
+	size_t nd = datasize - datacount;
+	//fprintf(stderr, "t numElems %lu datacount %lu nn %lu nd %d datasize %lu \n", numElems, datacount, nn, nd, datasize);
+	for (size_t n = nd; n < datasize && nn > 0; ++n) {
+		*out++ = datasave[2 * n + 0];
+		*out++ = datasave[2 * n + 1];
+		--nn;
+		--datacount;
+	}
```

Comment:
this just dawned on me, but this and the other copy loop should really should just be a std::memcpy()/std::copy() without the loop and just incrementing the pointers/counts by a fixed amount.